_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <footer>
   <a class="logo" href="/" rel="author">
-    <img alt="Logo" height="57" src="/assets/images/logo.png" width="253"/>
+    <img src="/assets/images/logo.svg" width="256" height="57" alt="Mangata" />
```

Comment:
you can use **require("/assets/images/logo.svg").default** in src .. I read somewhere that it is better way to load images.

Reply:
The blog doesn't use webpack and/or react. I see your point that putting images as base64 will display them faster than loading them via requests, however that's not really possible here.

Reply:
ok  great :)